src/quicktype-graphql-input/index.ts

```diff
@@ -269,6 +273,46 @@ class GQLQuery {
         return fragment;
     };
 
+    private InputObjectDepth: number = 0;
```

Comment:
Our schema has an input type which refers to itself. Unfortunately the properties are generated before we have a reference before the type, resulting in an infinite recursion. This was my hacky way around the issue without refactoring a ton of code. This could be removed if one of two things were supported
1. The delayed loading of types
2. The ability to change properties on a type after it had been created
I did not see a way to pull either off without a ton of edits